docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+
+## 3. **Time Period**
+- **Definition:** The temporal component of the grid cell. This could be a month, year, or any other defined period.
+- **Example:** A time period could be a month when measuring monthly fatalities per 100k population.
+
+## 4. **Potential Event**
+- **Definition:** Any grid cell within the defined geographic and temporal scope that could potentially contain an event.
+- **Example:** If a country is divided into 100 grid cells and observed over 400 months, there are 40,000 potential events.
+
+## 5. **Cumulative Distribution Function (CDF)**
+- **Definition:** A function that represents the cumulative probability of observing a value of a certain magnitude or lower. In this context, it shows the probability of seeing a certain number of fatalities per 100k population or less in any grid cell.
+- **Example:** A CDF value of 0.7 at 10 fatalities per 100k means there's a 70% chance of observing 10 or fewer fatalities per 100k in a random grid cell.
+
+## 6. **Event Probability (p_i)**
+- **Definition:** The probability of observing an event of a specific size or larger in a single grid cell, derived from the CDF. It is essentially the complement of the CDF at a given value, representing the probability of exceeding that value.
+
+- **Formula:** 
+$$p_i = 1 - \text{CDF}(\text{value})$$
+
+- **Example:** If the CDF at 10 fatalities per 100k is 0.7, then $p_i$ is 0.3, meaning there's a 30% chance of observing more than 10 fatalities per 100k in a random grid cell.
+
+## 7. **Event Return Period (e_i)**
+- **Definition:** The expected number of grid cells (trials) that need to be observed to see at least one event of a specific size or larger. If $p_i$ is the event probability, we have:
+
+- **Formula:** 
+
+$$e_i = \frac{1}{p_i}$$ 
+
+- **Example:** If $p_i$ is 0.1, the event return period $e_i$ is 10, meaning you need to observe 10 grid cells on average to see an event of that size or larger.
+
+## 8. **Time-Based Event Return Period (E_i)**
+- **Definition:** The expected number of time periods (e.g., months, years) between occurrences of a specific event of a certain size or larger in a defined geographic area. The term is general and can be adapted to different time scales, such as the Annual Event Return Period ($E^a_i$) for years or the Monthly Event Return Period ($E^m_i$) for months. If $P_i$ is the probability of at least one occurrence in the given area and time period, we have:
+
+- **Formula:** $$E_i = \frac{\text{Number of Time Periods}}{P_i}$$
+
+
+
+- **Example (Annual):** If $P_i$ is 0.1 and you have 100 years of observation, $E^a_i$ would be 10 years, meaning the event is expected to occur once every 10 years.
+
+- **Example (Monthly):** If $P_i$ is 0.1 and you have 100 months of observation, $E^m_i$ would be 10 months, meaning the event is expected to occur once every 10 months.
+
+## 9. **At Least One Occurrence Probability (P_i)**
```

Comment:
Consider moving point 9 above point 8 because it is referenced for the first time in point 8.